pkg/scheduler/api/types.go

```diff
@@ -192,6 +192,9 @@ type ExtenderConfig struct {
 	//   will skip checking the resource in predicates.
 	// +optional
 	ManagedResources []ExtenderManagedResource
+	// Ignorable specifies if the extender is ignorable, i.e. scheduling should not
+	// fail when this extender returns an error.
```

Comment:
To make things more clear for the users, please add to the comment that:
"when the extender returns an error **or is not reachable**."